chemprop/cli/predict.py

```diff
@@ -261,9 +262,10 @@ def main(args):
         "preds"
     ] = preds.flatten()  # TODO: this will not work correctly for multi-target predictions
     if args.output.suffix == ".pkl":
-        df_test.to_pickle(args.output, index=False)
+        df_test.to_pickle(args.output)
```

Comment:
Why do we keep the index?

Reply:
`to_pickle` doesn't have `index` as an arguement. I'll add `df_test = df_test.reset_index(drop=True)` so the index can be ignored.